.gitignore

```diff
@@ -1,40 +1 @@
-COPYING
-INSTALL
-README
-AUTHORS
-NEWS
-Makefile.in
-Makefile
-aclocal.m4
-autom4te.cache
-configure
-config.cache
-config.h
-config.log
-config.rpath
-config.status
-compile
-depcomp
-description-pak
-install-sh
-missing
-po/stamp-it
-src/gamesxml2c
-.*.swp
```

Comment:
https://news.ycombinator.com/item?id=1688068

There is no place in tree for such garbage to start with.